Wireframe/index.html

```diff
@@ -1,13 +1,63 @@
+
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css" integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    
+
+    <img class="git-logo" src="photos/logo@2x.png" alt="">
+     <p id="index-p"> Git is the most commonly used version control system</p>
+     <div class="image">
+     <img src="photos/branching-illustration@2x.png" alt=" git " width="1200px" height="300px">
+    </div>
+     <div class="git-info">
+   
+     <h2>what is Git</h2>
+     
+     
+     <p class="first-p">Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. Git is easy to learn and has a tiny footprint with lightning fast performance. It outclasses SCM tools like Subversion, CVS, Perforce, and ClearCase with features like cheap local branching, convenient staging areas, and multiple workflows. <a  class="links">read more</a></p>
+     </div>
+
+   <div class="p1">
+    <h1>Why do developers need Git</h1>
+    <img src="photos/gitt.png" alt=""  width="400px" >
+    <p class="Paragraph"> Git is all about efficiency. For developers, it eliminates everything from the time wasted passing commits over a network connection to the man hours required to integrate changes in a centralized version control system. It even makes better use of junior developers by giving them a safe environment to work in <a class="links" href="https://www.quora.com/Why-does-a-front-end-developer-need-to-learn-git-Atleast-I-see-multiple-sites-that-says-so">read more</a></p>
+    
+  </div>
+   <div class="p2">
+    <h1 >What is a branch in Git</h1>
+
+    <img src="photos/Untitled.png" alt=" " width="500px">
+    
+    <p class="Paragraph">branches are a part of your everyday development process. Git branches are effectively a pointer to a snapshot of your changes. When you want to add a new feature or fix a bug—no matter how big or how small—you spawn a new branch to encapsulate your changes.<a class="links" href="https://codeinstitute.net/blog/git-branches/">read more</a></p>
+    
+   
+   </div>
+  
+   <hr style="width:90%;text-align:right;margin-left:20">
```

Comment:
Please, avoid inline styling. If not inline, what is other way to style this horizontal line?

Reply:
I will try to fix it thanks